### PR TITLE
Support Partitioned Cookies

### PR DESCRIPTION
**Pull request by Gaurav0**
Adds support for Partitioned Cookies. Useful for third party cookies. See https://developer.mozilla.org/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies

### CODE REVIEW

File: packages/ember-cookies/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cookies",
-  "version": "1.1.2",
```
Comment (BobrImperator) [inline packages/ember-cookies/package.json:3]: Version shouldn't be changed. It's automatically incremented during release.

**Review by BobrImperator — COMMENTED** [review]
Lockfile likely was used with a different pnpm version.

**Comment by Gaurav0** [comment]
@BobrImperator updated.

File: packages/ember-cookies/package.json (hunk 1 above)
Comment (Gaurav0) [reply packages/ember-cookies/package.json]: I updated the version to remind you that this is a new feature and as such merits a minor version update. Removing the version bump.

**Comment by Gaurav0** [comment]
@BobrImperator rebased.

**Comment by Gaurav0** [comment]
@BobrImperator rebased

**Pull request merged**